client/src/components/Onboarding.js

```diff
@@ -68,9 +68,44 @@ function ProgressMobileStepper({activeStep}) {
     );
 }
 
-function Onboarding({children,title,activeStep,path}) {
+function Onboarding({children,title,activeStep,path,url}) {
 
     const classes = useStyles();
+    
+    const history = useHistory();
+
+    const [urlStore, setUrlStore] = React.useState("");
```

Comment:
This is for restoring the url from parent `ProfileSettings` to `Onboarding` state, so that in the final step, `Onboarding` will be able to collect all the date need to send to BE